articles/cognitive-services/Acoustics/faq.md

```diff
@@ -67,9 +67,9 @@ Probes are a sampling of possible player locations. Each probe represents a sepa
 
 Project Acoustics provides accurate and reliable acoustic parameters even for ultra-complex virtual environments, taking every architectural aspect into account. It provides smooth occlusion and obstruction and dynamic reverb variation without the manual work of drawing volumes. All while remaining light on CPU during runtime.
 
-## What exactly happens during "baking"?
+## What exactly happens during "baking"? Is my source content secure?

```

Comment:
I think this should be two separate questions. The security aspect seems like it deserves separate emphasis.

Reply:
Fine by me. I'll update.